stats/stats_test.go

```diff
@@ -81,6 +81,50 @@ var (
 	}
 	// The id for which the service handler should return error.
 	errorID int32 = 32202
+	// To verify if the Unary RPC server stats events are logged in the
+	// correct order.
+	expectedUnarySequence = []string{
+		"ConnStats",
+		"InHeader",
+		"Begin",
+		"InPayload",
+		"OutHeader",
+		"OutPayload",
+		"OutTrailer",
+		"End",
+	}
+	// To verify if the Client Stream RPC server stats events are logged in the
+	// correct order.
+	expectedClientStreamSequence = []string{
+		"ConnStats",
+		"InHeader",
+		"Begin",
+		"OutHeader",
+		"InPayload",
+		"InPayload",
+		"InPayload",
+		"InPayload",
+		"InPayload",
+		"OutPayload",
+		"OutTrailer",
+		"End",
+	}
+	// To verify if the Server Stream RPC server stats events are logged in the
+	// correct order.
+	expectedServerStreamSequence = []string{
```

Comment:
same. can be moved to respective test. s/expectedServerStreamSequence/wantServerStreamSequence